src/settings/Settings.js

```diff
@@ -188,6 +188,11 @@ export const SETTINGS = {
         default: true,
         invertedSettingName: 'MessageComposerInput.dontSuggestEmoji',
     },
+    // TODO: Wire up appropriately to UI (FTUE notifications)
+    "Notifications.alwaysShowBadgeCounts": {
+        supportedLevels: ['account'],
```

Comment:
The actual badge behaviour and scope is largely something for another PR. This PR is more meant to just try and get *something* in front of people to poke at, with iteration on the new (complicated) rules coming in a future PR.